content/adventures/en.yaml

```diff
@@ -790,11 +790,13 @@ adventures:
                     {ask} _
                     {echo}
                     {echo}
-                start_code: |
+                start_code: |-
+                    ```
```

Comment:
@Mark-Giesen start_code should not be surrounded by ``` (it is the code that is shown in the editor) it is not in other places also. What prompted this change?

Reply:
mmm, haste maybe? Not sure. Remember doing it, but not why. I'll be translating more this week, I'll be cautious.

Reply:
No problem, mistakes happen (this is why we have tests!) Can you put it back the way it was in Weblate? That is easier that reverting the commit here.